### DISCUSSION TITLE
Issue with plugins

### DISCUSSION THREAD

**Discussion by anvy2** [outline/rich-markdown-editor#429] [Q&A] [answered]
I was trying to make a plugin to use with this editor. Checked the existing plugins folder in repo and used it as reference but i'm getting an error 

TypeError
extension.bindEditor is not a function

This is the sandbox for it
https://codesandbox.io/s/optimistic-almeida-8h5vf?file=/src/App.js

**Comment by tommoor** [comment, marked as answer]
You need to instantiate an instance of the class:

https://codesandbox.io/s/wizardly-paper-h8d9b?file=/src/App.js

**Reply by anvy2** [reply]
Got it. Thanks a lot
